app/constants/AppConstants.js

```diff
@@ -8,7 +8,7 @@ export default {
     'varaamotest-espoo.hel.ninja': 'ESPOO',
     'varaamo.vantaa.fi': 'VANTAA',
     'varaamotest-vantaa.hel.ninja': 'VANTAA',
-    'varaamo.tampere.fi': 'TAMPERE', // PROD
+    'localhost:3000': 'TAMPERE', // PROD
```

Comment:
We don't want to change this. This would default to using Helsinki specific page for Varaamo.